client/ayon_max/mxp.py

```diff
@@ -0,0 +1,48 @@
+import os
+from ayon_core.lib import Logger
+
+
+def create_workspace_mxp(workdir, mxp_workspace=None):
+    dst_filepath = os.path.join(workdir, "workspace.mxp")
+    if os.path.exists(dst_filepath):
+        return
+
+    log = Logger.get_logger("create_workspace_mxp")
+    os.makedirs(workdir, exist_ok=True)
+    max_script = default_mxp_template()
+    if mxp_workspace.get("enabled_project_creation"):
```

Comment:
This will error if `mxp_workspace` is not provided and hence is `None`.

```suggestion
    if mxp_workspace and mxp_workspace.get("enabled_project_creation"):
```